### PR TITLE
Make 'build-type' optional with default value 'dailybuild'

### PR DESCRIPTION
**Pull request by rshym**

### CODE REVIEW

**Comment by rshym** [comment]
Please see below part of console output without build-type specified.
```
rsm@pc2:~/projects/build-scripts$ time python ./build_prod.py --machine salvator-xs-h3 --product tu2019-demo --branch master --config rs_tu.cfg  --with-local-conf
Removing /mnt/data1t/work_tu/build
Removing /mnt/data1t/work_tu/tmp/current-build-cache
Type: dailybuild product: prod-tu2019-demo machine: salvator-xs-h3
Not using build history
```

**Comment by iusyk** [comment]
I am ok

File: README [hunk 1]
```diff
@@ -31,6 +21,17 @@ Required paramters
 Optional arguments
 ==================
 
+--build-type - specifies build type to perform:
+  * dailybuild        - perform the build based on HEAD revisions of meta layers,
+                        e.g. this is the latest snapshot.
+                        This is default behaviour if build-type is not specified.
```
Comment (andr2000) [inline README]: the default

**Comment by andr2000** [comment]
With the minor wording fix:
`Reviewed-by: Oleksandr Andrushchenko <oleksandr_andrushchenko@epam.com>`

**Comment by arminn** [comment]
LGTM

**Pull request merged**